crates/pop-common/src/rust_writer_helpers/sample_files/basic_pallet.rs

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
To keep things simpler and avoid introducing extra new code (even if it's just sample files for testing), would it be possible to just instantiate a new pallet in a temporary file instead?

For reference, we follow a similar approach for parachain testing here: https://github.com/r0gue-io/pop-cli/blob/main/crates/pop-parachains/src/new_parachain.rs#L102